### PR TITLE
bpo-35450: reflect in docs that venv module is not always creating a …

### PR DESCRIPTION
**Pull request by mkkot**
…copy of python binary

https://bugs.python.org/issue35450

### CODE REVIEW

File: Doc/using/venv-create.inc
Comment (brettcannon) [inline Doc/using/venv-create.inc]:
```suggestion
containing a copy/symlink of the Python binary/binaries (as appropriate for the
```

**Review by brettcannon — CHANGES_REQUESTED** [review]

**Comment by brettcannon** [comment]
@mkkot only one minor grammatical change to make. Otherwise I'm just waiting for your CLA to be processed and then I can merge the change!

**Comment by mkkot** [comment]
Sure, please merge. What about backporting, who is dealing with this?

**Comment by brettcannon** [comment]
@mkkot our magical bots handle backports 😄  (that's what the labels actually trigger).

**Comment by brettcannon** [comment]
@mkkot I can't merge until you either accept the suggested fix or click the checkbox to let maintainers modify your branch (I just tried and GitHub says I can't do it).

**Comment by mkkot** [comment]
@brettcannon I resolved the conversation, hope now you can merge.

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
@mkkot I actually wanted the suggested change to get merged, but it turned out I had the appropriate permissions to edit the file, so I just went ahead and did that.

Thanks for the PR!

**Pull request merged**
